### PR TITLE
Sponsorship update

### PR DESCRIPTION
**Pull request by daveverwer**
Changes:

* README now has a list of all sponsor logos that link to the sponsors, followed by a bulleted list of sponsors with slightly shorter text. Text not yet confirmed by sponsors.
* Community sponsors are now highlighted better in the README.
* Sponsors are now both shown on the home page, but with the slightly shorter text. The order of sponsors is randomised.
* A selection of random community sponsor avatars are now shown on the home page.

Screenshot 2022-08-31 at 15 10 53@2x
Screenshot 2022-08-31 at 15 11 08@2x

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,41 +2,38 @@
 
 **The Swift Package Index helps you make better decisions about the dependencies you use in your apps**.
 
-The [Swift Package Index](https://swiftpackageindex.com) is a search engine for packages that are compatible with the [Swift Package Manager](https://swift.org/package-manager/). The project is open-source and written in Swift using [Vapor](https://swiftpackageindex.com/vapor/vapor).
+The [Swift Package Index](https://swiftpackageindex.com) is tha place to find packages that are compatible with the [Swift Package Manager](https://swift.org/package-manager/). The project is [open-source](https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/blob/main/LICENSE) and written in Swift using [Vapor](https://swiftpackageindex.com/vapor/vapor).
```
Comment (finestructure) [inline README.md:5]: it's da place mon :D

Comment (finestructure) [inline README.md:5]:
```suggestion
The [Swift Package Index](https://swiftpackageindex.com) is the place to find packages that are compatible with the [Swift Package Manager](https://swift.org/package-manager/). The project is [open-source](https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/blob/main/LICENSE) and written in Swift using [Vapor](https://swiftpackageindex.com/vapor/vapor).
```

**Review by finestructure — APPROVED** [review]
Looks great!

**Comment by daveverwer** [comment]
Deploying as 2.75.7

**Pull request merged**